src/types.ts

```diff
@@ -18,12 +18,12 @@ interface IWalletStoreState {
   address: string | null
   addressShort: string | null
   addressDomain: string | null
-  balance: string | null
+  balance: string | number | null | undefined
```

Comment:
`string | number | null | undefined` looks inconsistent.
Is it easy to make types at least `balance: string | null`?
(of course, better `balance: number | null` / `balance: BigNumber | null`)

Reply:
Yo, if we didn't have these features with `any`, [like this](https://github.com/viaprotocol/web3-wallets/blob/367285760a9358f8f5ce43668de5bde62cb9ff31/src/context/WalletProvider.tsx#L30-L31), there would be a lot less problems :)

Reply:
Outdated